pubspec.yaml

```diff
@@ -11,7 +11,10 @@ dependencies:
   flutter:
     sdk: flutter
   flutter_compass: ^0.6.1
-  flutter_map: ^0.13.1
+  flutter_map: 
+    git: 
+      url: "https://github.com/sandeshgyw/flutter_map.git"
```

Comment:
Could you achieve the same result my using the released version of `flutter_map_location` but override the `flutter_map` using `depencency_override` in your app's `pubspec.yaml`?